extensions-modules/src/languageservice/serviceClient.ts

```diff
@@ -351,7 +351,9 @@ export class SqlToolsServiceClient {
 							name: '',
 							show: () => {
 							}
-						}
+						},
+						// pass in no features so we don't register features we don't want
+						features: []
```

Comment:
where is the change so that mssql will use the defaultFeatures array?  or is that already setup somewhere?

Reply:
If you don't provide teh features option, it defaults to registering everything, so by changing this to a empty array, it won't register any features (what we want) but the place where the main service is register I don't specify a features argument, so it defaults to registering everything.